api/src/test/java/org/openmrs/api/db/hibernate/HibernateOrderDAOTest.java

```diff
@@ -173,4 +177,92 @@ public void deleteOrderGroupAttributeType_shouldDeleteOrderGroupAttributeTypeFro
 		dao.deleteOrderGroupAttributeType(orderGroupAttributeType);
 		assertNull(dao.getOrderGroupAttributeByUuid(UUID));
 	}
+
+	/**
+	 * @see {@link HibernateOrderDAO#getAllOrderAttributeTypes()}
+	 * @throws Exception
+	 */
+	@Test
+	public void getAllOrderAttributeTypes_shouldGetAllOrderAttributeTypes() {
+		List<OrderAttributeType> orderAttributeTypes = dao.getAllOrderAttributeTypes();
+		assertEquals(orderAttributeTypes.size(), 4);
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getOrderAttributeTypeByUuid(String)}
+	 * @throws Exception
+	 */
+	@Test
+	public void getOrderAttributeTypeByUuid_shouldGetOrderAttributeTypeByUuidGivenTheUuid() {
+		OrderAttributeType orderAttributeType = dao.getOrderAttributeTypeByUuid("9758d106-79b0-4f45-8d8c-ae8b3f25d72a");
+		assertEquals("Referral", orderAttributeType.getName());
```

Comment:
you don't need line 197 ...You could do it like . Since usage is in one place only
`assertEquals("Referral", dao.getOrderAttributeTypeByUuid("9758d106-79b0-4f45-8d8cae8b3f25d72a").getName());`